features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_timer.cpp

```diff
@@ -47,6 +47,7 @@ void platform_timer_enable(void)
 #endif
     // Prime the SingletonPtrs - can't construct from IRQ/critical section
     timer.get();
+    timer->start();
```

Comment:
Um. Seriously? I guess no-one was ever relying on this in the absence of frequency-hopping?

Yikes.

Anyway, if you're doing `timer->start()`, don't need the "dummy" `timer.get()`, so remove that line.

Reply:
Oh, and maybe put the "prime" comment above just the remaining timeout.get, as it's about that dummy get. timer->start() is inherently obvious.

But apparently not obvious enough for me to put in the code in the first place. Sigh.

Reply:
@kjbracey-arm I guess, even without frequency hopping it caused some issues with MAC timer. Too short Ack wait durations caused MAC retransmissions etc. Please check the changes I made.